src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1324,8 +1324,20 @@ public function setRequestFormat(?string $format)
 
     /**
      * Gets the format associated with the request.
+     *
+     * @deprecated since Symfony 6.1, use getContentTypeFormat instead
      */
     public function getContentType(): ?string
+    {
+        trigger_deprecation('symfony/http-foundation', '6.1', 'The method "%s" is deprecated, use "getContentTypeFormat" instead.', __METHOD__);
+
+        return $this->getContentTypeFormat();
+    }
+
+    /**
+     * Gets the format associated with the request.
+     */
+    public function getContentTypeFormat(): ?string
```

Comment:
I agree with what this aims to fix, but the difference between `getRequestFormat()` and `getFormat()` was already not obvious before this PR. Adding `getContentTypeFormat` makes the API even more confusing. We should probably make the difference clear with PHPDoc at least